src/layouts/Base.astro

```diff
@@ -30,6 +30,7 @@ if (match && i18next.languages.includes(match[1])) {
 	/>
 
 	<script src="https://betteruptime.com/widgets/announcement.js" data-id="153552" async="async" type="text/javascript"></script>
+	<script src="https://challenges.cloudflare.com/turnstile/v0/api.js" async defer></script>
```

Comment:
Inconsistency between xhtml-style and html-style tags (with the implicit `async="async"` in the added line, ditto for `defer`)

Reply:
Those are using script tags as requested by each vendors